sympy/physics/vector/functions.py

```diff
@@ -593,19 +601,35 @@ def dynamicsymbols(names, level=0):
     >>> q1 = dynamicsymbols('q1')
     >>> q1
     q1(t)
-    >>> diff(q1, Symbol('t'))
+    >>> diff(q1, Symbol('t', real=True))
+    Derivative(q1(t), t)
+    >>> q1d = dynamicsymbols('q1', level=1)
     Derivative(q1(t), t)
 
+    Notes
+    =====
+
+    The default time symbol is stored on the function, i.e.::
+
+        >>> dynamicsymbols.t
+        t
+
+    It can be changed by simply overriding the attribute::
+
+        >>> dynamicsymbols.t = symbols('T', real=False)
+
     """
 
-    esses = symbols(names, cls=Function)
-    t = dynamicsymbols._t
-    if iterable(esses):
-        esses = [reduce(diff, [t] * level, e(t)) for e in esses]
-        return esses
-    else:
-        return reduce(diff, [t] * level, esses(t))
+    kwargs.pop('cls', None)
 
+    syms = symbols(names, cls=Function, real=real, **kwargs)
+
+    t = dynamicsymbols.t
+
+    if iterable(syms):
+        return tuple([reduce(diff, [t] * level, s(t)) for s in syms])
+    else:
+        return reduce(diff, [t] * level, syms(t))
 
-dynamicsymbols._t = Symbol('t')
+dynamicsymbols.t = Symbol('t', real=True)
```

Comment:
We should leave `_t` in place as an alias for backwards compat. I have lots of code that accesses `_t` that will break with this.